server/src/resolvers.js

```diff
@@ -4,16 +4,150 @@ const {User} = require('./resolvers/user');
+          creationTime: '',
+          lastUpdatedTime: '',
+          schedule: [],
+          title: 'Introduction to Algorithm',
+          description: 'Basic to ICPC World Final... You think you can handle this?',
+          price: 999999.99,
+          maxParticipants: 1000,
+          tags: [],
+          participants: []
+        },
+        {
+          id: 2,
+          host: 0,
+          creationTime: '',
+          lastUpdatedTime: '',
+          schedule: [],
+          title: 'What is meaning of life?',
+          description: 'Sometimes we thinking about huge size of the universe. Compare to what humanity accomplished so far, the universe seems wast of space... isn\' it?',
+          price: 999999.99,
+          maxParticipants: 1000,
+          tags: [],
+          participants: []
+        },
+        {
+          id: 3,
+          host: 0,
+          creationTime: '',
+          lastUpdatedTime: '',
+          schedule: [],
+          title: 'If AI gets self awareness...',
+          description: 'That means bad thing to human? YES. But in a perspective of evolution? NO.',
+          price: 999999.99,
+          maxParticipants: 1000,
+          tags: [],
+          participants: []
+        },
+        {
+          id: 4,
+          host: 0,
+          creationTime: '',
+          lastUpdatedTime: '',
+          schedule: [],
+          title: 'When Space X going to launch a rocket to the Mars?',
+          description: 'Let\'s analyse the probability!',
+          price: 999999.99,
+          maxParticipants: 1000,
+          tags: [],
+          participants: []
+        },
+        {
+          id: 5,
+          host: 0,
+          creationTime: '',
+          lastUpdatedTime: '',
+          schedule: [],
+          title: 'title1',
+          description: 'description1',
+          price: 999999.99,
+          maxParticipants: 1000,
+          tags: [],
+          participants: []
+        },
+        {
+          id: 6,
+          host: 0,
+          creationTime: '',
+          lastUpdatedTime: '',
+          schedule: [],
+          title: 'title2',
+          description: 'description2',
+          price: 999999.99,
+          maxParticipants: 1000,
+          tags: [],
+          participants: []
+        },
+        {
+          id: 7,
+          host: 0,
+          creationTime: '',
+          lastUpdatedTime: '',
+          schedule: [],
+          title: 'title3',
+          description: 'description3',
+          price: 999999.99,
+          maxParticipants: 1000,
+          tags: [],
+          participants: []
+        },
+        {
+          id: 8,
+          host: 0,
+          creationTime: '',
+          lastUpdatedTime: '',
+          schedule: [],
+          title: 'title4',
+          description: 'description4',
+          price: 999999.99,
+          maxParticipants: 1000,
+          tags: [],
+          participants: []
+        }
+      ];
+
+      let ret = [];
+      if(!pageSize) {
```

Comment:
if pageSize is missing, shouldn't we return Math.min(events_list.size(), 9)?

In fact, you can compute the following, and remove the if else block below:

int max_size = Math.min(events_list.size(), pageSize);
max_size = Math.min(max_size, 9)
ret = events_list.slice(0, max_size)